noxfile.py

```diff
@@ -1870,7 +1870,7 @@ def ci_test_onedir_pkgs(session):
 
     # Install requirements
     if _upgrade_pip_setuptools_and_wheel(session, onedir=True):
-        _install_requirements(session, "zeromq")
+        _install_requirements(session, "pyzmq")
```

Comment:
```suggestion
        _install_requirements(session, "pyzmq")
```
How did this ever work?!

Reply:
I have no idea.  My only guess is we were pulling deps from cache this whole time, but that doesn't seem likely.